docs/tutorials/08_qcschema.ipynb

```diff
@@ -0,0 +1,227 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3e8d47d1",
+   "metadata": {},
+   "source": [
+    "# QCSchema\n",
+    "\n",
+    "The [QCSchema](https://github.com/MolSSI/QCSchema) is a standard data format for quantum chemistry. The current version of it which has been adopted by multiple classical chemistry codes, only supports serialization via `JSON`, but their docs clearly indicate that `HDF5` may also be used. Thus, in Qiskit Nature, we have opted for implementing support for both hierarchical data formats.\n",
+    "\n",
+    "Now, why do we have a tutorial specifically about this format you may wonder? The reason is fairly simple: this is the data format which our drivers use internally to transfer data between the classical computation and Qiskit Nature. Thus, this tutorial will explain a few concepts, in case you want to get a bit more elaborate with your driver interaction.\n",
+    "\n",
+    "> **Note:** the support for electronic-repulsion integrals as part of the QCSchema is not yet part of the official specification and, thus, custom to Qiskit Nature. But we are working with the QCSchema authors to make this integration official!\n",
+    "\n",
+    "For the purposes of this tutorial, we are using the `PySCFDriver`, but most discussion points should apply to the other electronic structure drivers, too.\n",
+    "\n",
+    "First, let us construct a `PySCFDriver` and run it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "58fc6126",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<qiskit_nature.second_q.problems.electronic_structure_problem.ElectronicStructureProblem object at 0x155041be1250>\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.drivers import PySCFDriver\n",
+    "\n",
+    "driver = PySCFDriver()\n",
+    "\n",
+    "problem = driver.run()\n",
+    "print(problem)"
```

Comment:
Does this not print anything? - I do not see any output (aside from what might be an extra blank line) - maybe you edited the cell and added this and did not re-run it.

Reply:
Right now these just print
```
<qiskit_nature.second_q.problems.electronic_structure_problem.ElectronicStructureProblem object at 0x155041be1250>
```
which should also show in Sphinx rendered version. While not super insightful, it at least reflects the output type that's why I left these in.

In the future I would like to add some string-representations of problems similar to what we had for the properties and then the tutorials will be able to benefit from this directly :+1: